ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/remove_unnecessary_columns.md

```diff
@@ -0,0 +1,56 @@
+# DF: Remove Unnecessary Column(s)
+
+## Use case
+
+You can use it to remove specified column(s).
+
+## Prerequisites
+Follow the [guide](../README) and setup your policies correctly.
```

Comment:
Pls add a line: You can use utility-meter-readings(link to the csv in sample datasets) as an example